test/functionality/dataflow/environments/resolve-tests.ts

```diff
@@ -42,4 +45,33 @@ describe('Resolve', () => {
 			});
 		});
 	});
+	describe('Builtin Constants', () => {
+		const testResolve = (label: string, identifier: Identifier | undefined, wantedValue: unknown, expectedResult: Ternary, environment: REnvironmentInformation = defaultEnv()) => it(label, () => {
+			const result = resolvesToBuiltInConstant(identifier, environment, wantedValue);
+			assert.strictEqual(result, expectedResult, `should be Ternary[${expectedResult}]`);
+		});
+
+		//			Label				    Identifier	Wanted Value  Expected Return Value 
+		testResolve('Resolve TRUE',		    'TRUE', 	true,		  Ternary.Always);
+		testResolve('Resolve TRUE',		    'TRUE', 	true,		  Ternary.Always);
+		testResolve('Resolve T',		    'T', 		true,		  Ternary.Always);
+		testResolve('Resolve FALSE',	    'FALSE',	false,		  Ternary.Always);
+		testResolve('Resolve F',		    'F',		false,		  Ternary.Always);
+		testResolve('Resolve NULL',		    'NULL', 	null,		  Ternary.Always);
+		testResolve('Resolve NA',		    'NA', 		null,		  Ternary.Always);
+
+		testResolve('Maybe Resolves TRUE',  'TRUE', 	true,		  Ternary.Maybe, 
+			defaultEnv().defineInEnv({ name: 'TRUE', nodeId: 0, definedAt: 1, type: ReferenceType.Constant, controlDependencies: [{ id: 42, when: true }] })
+		);
+		testResolve('Maybe Resolves FALSE', 'FALSE', 	false,		  Ternary.Maybe, 
```

Comment:
From a technical perspective this is probably redundant, aber let's keept it this way :)